### PR TITLE
[INCR-98] and [INCR-99] Run Python modernize -w .

### PR DESCRIPTION
**Pull request by cclauss**
Fixes https://openedx.atlassian.net/browse/INCR-98, https://openedx.atlassian.net/browse/INCR-99

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @cclauss! I've created [OSPR-3125](https://openedx.atlassian.net/browse/OSPR-3125) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: tox.ini [hunk 1]
```diff
@@ -33,4 +33,4 @@ deps =
 
 commands =
     pylint workbench sample_xblocks
-    isort --check-only --recursive workbench sample_xblocks manage.py setup.py settings.py
+    # isort --check-only --recursive workbench sample_xblocks manage.py setup.py settings.py
```
Comment (estute) [inline tox.ini]: @cclauss why are you commenting this out here?

**Comment by natabene** [comment]
@cclauss Thank you for your contribution. @edx/incr-reviewers Would you like to review this?

**Comment by cclauss** [comment]
The commits were:
1. Run python-modernized
2. Placate pylint
3. Run isort
4. Disable isort

The Travis error at the end of commit 2 was an isort complaint so in commit 3, I ran isort and it shuffled the imports around as it does but then complained again that the imports were improperly sorted.  Given that isort does not agree with isort, I made commit 4.

**Comment by estute** [comment]
@cclauss we would rather keep the isort check in place. Can you give some more fine-grained details about what was actually failing so we can help out?

**Comment by cclauss** [comment]
https://travis-ci.org/edx/xblock-sdk/jobs/497524653#L311

**Comment by jmbowman** [comment]
Try adding the `--diff` parameter to isort to get more useful output from it, like in https://github.com/edx/cookiecutter-django-app/blob/master/tox.ini#L31 .  (We found that option a little while after we started using isort, and haven't gone back to update everything to use it yet.)

**Comment by cclauss** [comment]
@estute I think we are set now...

**Comment by edx-webhook** [comment]
@cclauss 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/xblock-sdk&entry.867055334=https://github.com/edx/xblock-sdk/pull/147&entry.1484655318=https://github.com/cclauss&entry.752974735=2019-02-23+17:00&entry.1917517419=2019-02-27+01:47&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**